pyscript.core/esm/runtime/_python.js

```diff
@@ -0,0 +1,27 @@
+import { clean, writeFile as writeFileUtil } from "./_utils.js";
+
+// REQUIRES INTEGRATION TEST
+/* c8 ignore start */
+export const run = (runtime, code) => runtime.runPython(clean(code));
+
+export const runAsync = (runtime, code) => runtime.runPythonAsync(clean(code));
+
+export function runEvent(runtime, code, key) {
+    code = `import js;event=js.__events.get(${key});${code}`;
```

Comment:
Looks like this code just moved in this PR but ideally you should avoid adding a js global by using `registerJsModule`.

Reply:
What I was imagining was `from jsevents import __events; event = __events.get(whatever)`. This would work the same but remove one js global variable.

Reply:
I've been explicitly asked if that was needed and I've answered "for the time being no" ... so I think it got removed, I haven't even tried that's the case ... if it's not, I'll switch to your approach if that works better at the WASM / Python level.